src/basic-syntax/static-and-const.md

```diff
@@ -36,4 +38,15 @@ fn main() {
 }
 ```
 
+As noted in the [Rust RFC Book][1], these are not inlined upon use and have an actual associated memory location.  This is useful for unsafe and embedded code, and have a `'static` lifetime[^lifetime].
```

Comment:
Could you move this sentence to the speaker notes as well?

I think the `[^lifetime]` footnote looks like a dangling pointer now? :smile:

Reply:
Let me clean that up real quick.